beginner_source/blitz/autograd_tutorial.py

```diff
@@ -1,198 +1,276 @@
 # -*- coding: utf-8 -*-
 """
-Autograd: Automatic Differentiation
-===================================
-
-Central to all neural networks in PyTorch is the ``autograd`` package.
-Let’s first briefly visit this, and we will then go to training our
-first neural network.
+A Gentle Introduction to Autograd
+---------------------------------
+
+Autograd is PyTorch’s automatic differentiation engine that powers
```

Comment:
Fix: The `torch.autograd` package is PyTorch's automatic differentiation engine

Context for Suraj: this is the real autograd package: https://github.com/HIPS/autograd , we used the name, somewhat accidentally. So, it's important to introduce `torch.autograd` or as the `autograd` pacakge, not just Autograd. Please adjust references in this tutorial to Autograd as needed after this context.